src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java

```diff
@@ -121,14 +120,21 @@ private NotificationResponse sendLetterNotification(Map<String, Object> template
         return notificationService.sendLetter();
     }
 
-    private void addDecisionsIssuedFileContents(CaseIssueDecision caseIssueDecision, Map<String, Object> templateVars) throws IOException {
+    private void addDecisionsIssuedFileContents(CaseData caseData, Map<String, Object> templateVars) throws IOException {
         int count = 0;
 
         final String authorisation = idamService.retrieveSystemUpdateUserDetails().getAuthToken();
         String serviceAuthorization = authTokenGenerator.generate();
 
-        if (caseIssueDecision.getDecisionDocument() != null) {
+        CicCase cicCase = caseData.getCicCase();
+        /*if (caseIssueDecision.getDecisionDocument() != null) {
             List<String> uploadedDocumentsUrls = caseIssueDecision.getDecisionDocument().stream().map(item -> item.getValue())
+                .map(item -> StringUtils.substringAfterLast(item.getDocumentLink().getUrl(), "/"))
+                .collect(Collectors.toList());*/
+
```

Comment:
Commented out code?

Reply:
Fergal, this code is temporary, I need to test using already saved document as per TODO comment. Will revert this code once i test this scenario in AAT env as i cant do in local env.